index.html

```diff
@@ -302,25 +302,31 @@
   
         <dt><dfn>addRange</dfn></dt>
         <dd>
-          <p>The method must set the <a>context object</a>'s range to <var>range</var> by a strong reference (not by making a copy)
-            if the <dfn><a href="http://www.w3.org/TR/dom/#concept-tree-root">root</a></dfn> ([[!DOM4]])
-            of the <var>range</var>'s boundary points are the <a>document</a> associated with <a>context object</a>.
-            Otherwise, this method must do nothing.</p>
-  
+          <p>The method must follow these steps:</p>
+          <ol>
+            <li>If the <dfn><a href="http://www.w3.org/TR/dom/#concept-tree-root">root</a></dfn> ([[!DOM4]])
+                of the <var>range</var>'s boundary points are not the <a>document</a> associated with <a>context object</a>,
+                abort these steps.</li>
+            <li>if <code>rangeCount</code> is <code>0</code>, set the <a>context object</a>'s range to <var>range</var>
```

Comment:
Please reverse this step so that when `rangeCount` is not 0, we abort these steps.
So that these two if's have a consistent semantics (both aborts when the condition is matched).